src/Tags/Tagging.hs

```diff
@@ -23,11 +23,11 @@ import           Tags.Taggable
 
 runTagging :: (IsTaggable syntax)
            => Language
-           -> Source.Source
            -> [Text]
+           -> Source.Source
```

Comment:
This parameter order change gives it better compositionality, I’m assuming?

Reply:
Yep.